openassessment/xblock/static/js/src/lms/oa_course_items_listing.js

```diff
@@ -151,27 +159,21 @@
         $section.find(".open-response-assessment-summary").empty();
 
         $.each(this._columns, function(index, v) {
-            var realName = v.name === 'waiting' ? 'staff' : v.name;
             summaryData.push({
                 title: v.label_summary,
                 value: 0,
                 num: v.num,
-                class: realName
+                class: v.name
             });
-            summaryDataMap[realName] = index;
+            summaryDataMap[v.name] = index;
 
         });
 
         $.each(data, function(index, obj) {
-            var staffGradeRequired = obj.staff_assessment;
             $.each(obj, function(key, value) {
                 var idx = 0;
-                var realKey = key;
-                if (key === 'waiting' && staffGradeRequired) {
-                    realKey = 'staff';
-                }
-                if (realKey in summaryDataMap) {
-                    idx = summaryDataMap[realKey];
+                if (key in summaryDataMap) {
+                    idx = summaryDataMap[key];
                     if (summaryData[idx].num) {
```

Comment:
Is it possible that this will result in unexpected behavior if `summaryData[idx].num` is `0` vs. `undefined`?  I guess I'm unsure if this `if` statement is intended to account for both of those cases.